erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md

```diff
@@ -0,0 +1,73 @@
+# Leave Policy Assignment
+
+> Introduced in Version 13
+
+Leave Policy Assignment in ERPNext is used to grant leaves to employees. To access Leave policy assignment, go to:
```

Comment:
@Anurag810 add more info here, how will the user know what this feature does and why was it created?

Reply:
not required.